content/millennium/r4/care-provision/care-team.md

```diff
@@ -0,0 +1,140 @@
+---
+title: CareTeam | R4 API
+---
+
+# CareTeam
+
+* TOC
+{:toc}
+
+## Overview
+
+The R4 CareTeam resource can be used to represent care teams.
+
+The CareTeam resource represents a patient’s care team members. Care team members or participants include practitioners (physicians, nurses, technicians, etc.), family members, friends, guardians, and the patient. The care team can be specific to an encounter or to the patient across all encounters (longitudinal).
+
+The following fields are returned if valued:
+
+* [Care Team id](http://hl7.org/fhir/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status (Active)](http://hl7.org/fhir/careteam-definitions.html#CareTeam.status){:target="_blank"}
+* [Category (Longitudinal or Encounter)](http://hl7.org/fhir/careteam-definitions.html#CareTeam.category){:target="_blank"}
+* [Name (Encounter-Level Only)](http://hl7.org/fhir/careteam-definitions.html#CareTeam.name){:target="_blank"}
+* [Subject](http://hl7.org/fhir/careteam-definitions.html#CareTeam.subject){:target="_blank"}
+* [Encounter (Encounter-Level Only)]( http://hl7.org/fhir/careteam-definitions.html#CareTeam.encounter){:target="_blank"}
+* [Participant](http://hl7.org/fhir/careteam-definitions.html#CareTeam.participant){:target="_blank"}
+  * [Role](http://hl7.org/fhir/careteam-definitions.html#CareTeam.participant.role){:target="_blank"}
+  * [Member](http://hl7.org/fhir/careteam-definitions.html#CareTeam.participant.member){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:care_team, :r4) %>
+
+
+## Search
+
+Search for CareTeams that meet supplied query parameters:
+
+    GET /CareTeam?:parameters
+
+_Implementation Notes_
+
+* The [CareTeam.activity] element is not supported and will not be returned. This means that the [CareTeam.activity.detail.status] and [CareTeam.activity.detail.prohibited] modifier elements will not be returned.
+
+* The description of the CareTeam is included in the [CareTeam.text.div] element as ASCII text.
```

Comment:
this implementation note seems applicable both to search and read. if it is, we'll probably want to move it up

Reply:
+1

Reply:
Removed Implementation Note section after discussion with @sonikrish . Came to a conclusion that Implementation Note section is not required as the text included, " The description of the CareTeam is included in the [CareTeam.text.div] element as ASCII text." doesn't serve the purpose of the section.
Commit: https://github.com/cerner/fhir.cerner.com/pull/470/commits/743b57306d7803dca473eb256c61982ec9c0ff62

Reply:
I think mentioning where the CareTeam description can be found is important to include, especially since CareTeam.description isn't a normal field on the API

Reply:
But we don't have CareTeam.description supporting as of now not on the text div also. I think this got copied while looking through care plan since in CarePlan we do have a exception that description is only visible on the div part but not on the body. 
So, we should not have this under Implementation notes for CareTeam. Let me know if I am getting something wrong or missing?
Same applies for the CareTeam.activity we don't have any such field

Reply:
got it, yep, we'll need to remove this if we don't return the info in question